### ISSUE TITLE
[Beta]Scrolling on Instagram is not smooth

### ISSUE THREAD

**Issue by srirambv** [brave/browser-ios#537]
Description: 
[Beta]Scrolling on Instagram is not smooth

**Did you search for similar issues before submitting this one?**
Yes

**Expected behavior:**
Scrolling should be smooth on Instagram

- Device (iPhone5, iPhone6s plus, iPad 3, ?):
iPhone6 (iOS 10)

- Brave Version:
 v1.2.17 (16.10.31.18)

- Steps to reproduce:
    1. Login to Instagram
    2. Scroll down past the initial loaded images
    3. The next set of loaded images doesn't scroll smoothly

** Website problems only: 
- did you check with Brave Shields down? Yes
- did you check in Puffin/Dolphin/iCab/Maxthon (UIWebView-based browsers)? Checked on Dolphin

- Screenshot if needed:

- Any related issues:
#475

**Comment by garvankeeley** [comment]
Is it Beta only? How did Dolphin behave when you checked it? The same I assume

**Comment by srirambv** [comment]
Yes this is on Beta. On Dolphin it was comparatively sooth scroll compared to Brave Beta

**Comment by garvankeeley** [comment]
I am trying to reproduce now, but I am not sure exactly what to look for. As I scroll to the bottom, I see a new set of cells created, then images load in. Looks the same in a few browsers.

**Comment by srirambv** [comment]
This is what is happening on v1.2.17 (16.11.01.17) . After the initial set is loaded it loads new set of cells and loads images. But after the first load is done it doesn't load the second time. Shows Loading more and gets stuck there and doesn't load new set 

![image](https://cloud.githubusercontent.com/assets/17010094/19924660/d47b78a6-a0e4-11e6-832c-baca935cf6f7.png)

**Comment by garvankeeley** [comment]
Does it seem worse on beta than on the current release?

**Comment by srirambv** [comment]
Yes current release is comparatively smooth compared to beta.
